auth/go.mod

```
@@ -11,7 +11,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.17.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.8.2
 	github.com/bradleyfalzon/ghinstallation/v2 v2.14.0
-	github.com/fluxcd/pkg/cache v0.4.0
+	github.com/fluxcd/pkg/cache v0.4.1-0.20250301154518-d48089d18d8b
```

```suggestion
	github.com/fluxcd/pkg/cache v0.5.0
```

After https://github.com/fluxcd/pkg/pull/877 is released